informatiemodel/rdf/ontologie.ttl

```diff
@@ -164,6 +187,9 @@ zvg:Bedenktijd a <http://www.w3.org/2002/07/owl#Class>, sh:NodeShape;
     rdfs:label "Bedenktijd";
     rdfs:subClassOf zvg:Afspraak
 .
+zvg:Bedrag a rdf:Datatype;
```

Comment:
Bedrag willen we als een object hebben, het is een kwantiteit waarvan je de waarde en het soort moet kunnen aangeven, in dit geval de som en de valuta. 
zie bijvoorbeeld http://qudt.org/schema/qudt/Quantity

Reply:
Niet mee eens. Het is een gestructureerd datatype en geen objecttype. ("Specifiek benoemd datatype dat de structuur van een gegeven beschrijft, samengesteld uit minimaal twee elementen die in samenhang betekenisvol zijn.") We gebruiken ieg in MIM hier geen quidt voor. Dit bedrag is het gestructureerde datatype voor een kenmerk van een objecttype. 
Misschien zou je het rdf:datatype  zelf wel kunnen relateren aan de qudt:Quantity.